core/src/main/scala/chisel3/Data.scala

```diff
@@ -407,6 +418,40 @@ abstract class Data extends HasId with NamedComponent with SourceInfoDoc {
     _bindingVar = target
   }
 
+  private[chisel3] def scope: Option[Context] = {
+    _binding match {
+      case Some(ChildBinding(p: Data)) => p.scope
+      case Some(t: ConstrainedBinding) => t.location.get.contextOpt
+      case Some(ViewBinding(_)) =>
+        // For now, adding id to instance identifier because dataview names leave their local scope, as they are now children of `ViewParent
```

Comment:
```suggestion
        // For now, adding id to instance identifier because dataview names leave their local scope, as they are now children of `ViewParent`
```